Cargo.toml

```diff
@@ -36,8 +36,12 @@ name = "kem_api"
 harness = false
 
 [dev-dependencies]
-criterion = { version = "0.3", "features" = [ "html_reports" ] }
+rand = { version = "0.8", features = ["default"] }
```

Comment:
Because rand is a normal dependency with `default-features=false`, I need to re-enable it for testing since I use the `thread_rng` which requires `std` (I disabled `rand`s `std` by default since we do not need it, we only need the traits)